Source/WebInspectorUI/UserInterface/Models/DOMNodeStyles.js

```diff
@@ -122,7 +122,7 @@ WI.DOMNodeStyles = class DOMNodeStyles extends WI.Object
     }
 
     // Public
-
+    get usedFonts() { return this._usedFonts; }
```

Comment:
NIT: ideally this would be moved to match the order/location of the member variable